bitcart/coins/btc.py

```diff
@@ -90,9 +91,12 @@ async def balance(self) -> dict:
         data = await self.server.getbalance()
         return {attr: convert_amount_type(data.get(attr, 0)) for attr in self.BALANCE_ATTRS}
 
+    async def _addrequest(self, *args: Any, **kwargs: Any) -> dict:
+        return await self.server.add_request(*args, **kwargs)  # type: ignore
+
     async def addrequest(
         self,
-        amount: Optional[Union[int, str]] = None,
+        amount: Optional[AmountType] = None,
         description: str = "",
         expire: Union[int, float] = 15,
```

Comment:
What is the unit of "expire"? maybe expire_in_minutes?

Reply:
Expire's unit is minutes. It might be good to clarify in docstring, right?

Reply:
sure. docstring is better